packages/custodyController/src/custody.ts

```diff
@@ -104,27 +104,26 @@ export class CustodyController {
     return custodyTypes;
   }
 
+  // TODO - Can probably be removed as that state is not being used anywhere
   setCustodianConnectRequest({
     token,
-    apiUrl,
-    custodianName,
     custodianType,
+    envName,
   }: {
     token: string;
-    apiUrl: string;
-    custodianName: string;
     custodianType: string;
+    envName: string;
   }): void {
     this.store.updateState({
-      custodianConnectRequest: { token, apiUrl, custodianName, custodianType },
+      custodianConnectRequest: { token, envName, custodianType },
     });
   }
 
+  // TODO - Can probably be removed as it's not being called anywhere
```

Comment:
Updated the method to use envName, but added TODO as this method can probably be removed.